runtime/src/accounts_db.rs

```diff
@@ -100,6 +100,9 @@ pub type SnapshotStorages = Vec<SnapshotStorage>;
 // Each slot has a set of storage entries.
 pub(crate) type SlotStores = HashMap<usize, Arc<AccountStorageEntry>>;
 
+type PurgedAccountSlots = HashMap<Pubkey, HashSet<Slot>>;
+type ReclaimedAccounts = HashMap<AppendVecId, HashSet<usize>>;
```

Comment:
how about `AppendVecOffsets`/`reclaimed_offsets` over the status-quo?

- this type itself doesn't necessarily imply anything about being `reclaimed`. It's just some shaped container type alias.
- imply the `reclaimed` implication by variable names.
- avoid inconsistency between `reclaim`/`removed`. `reclaim` is preferred because this is related to `handle_reclaim`. `remove` is a bit too generic term; also actually we don't `remove` anything as appendvecs are always append only. So `reclaim` sounds good for me for this conceptual/logical delete here.

Reply:
Fixed in latest commit! https://github.com/solana-labs/solana/pull/12606/commits/69092648864a17e4a21d5699a51b5c2b6b83c16e